src/digitalmarketplace/components/alert/_alert.scss

```diff
@@ -0,0 +1,29 @@
+@import "../../../govuk-frontend/tools/all";
+@import "../../../govuk-frontend/helpers/all";
+
+@include govuk-exports("digitalmarketplace/component/alert") {
+
+    .dm-alert {
+        @include govuk-responsive-padding(4);
+        @include govuk-responsive-margin(8, "bottom");
+        border: govuk-spacing(1) solid;
+    }
+    .dm-alert--success {
+        border-color: govuk-colour("green");
+    }
+    .dm-alert--error {
+        border-color: $govuk-error-colour;
+    }
+    .dm-alert--noticees {
+        border-color: govuk-colour("blue");
+    }
+    .dm-alert__title {
+        @include govuk-font(24);
```

Comment:
This should only be font 24 if there is a param.text or param.html. May I suggest creating another class called `.dm-alert__title--large` which increases the font size to 24. by default the title should be font size 19